circuit_knitting/forging/entanglement_forging_knitter.py

```diff
@@ -282,10 +302,29 @@ def __call__(
                     )
                 self._session_ids[i] = job_id
 
+        # Pack NaNs into the expval list in place of the HF bitstring results to
+        # ensure Schmidt matrix is the correct shape
```

Comment:
So the purposes of the NaNs is just to mark things that should not be factored into the calculation?

Reply:
Yes, we want to create a schmidt matrix of a specific shape, but we want the element corresponding to the HF bitstring (element [0, 0]) to be NaN, so we can just replace it with a hard-coded value later.

Once we do the kroneker products to create that Schmidt matrix from the experiment results, all these NaNs will collapse to a single NaN in schmidt_mat[0, 0], and we'll replace it later.